app/models/AppealType.scala

```diff
@@ -32,6 +32,9 @@ object AppealType extends Enumeration {
     case _ => "Appeal"
   }
 
+  /**
+   * The order of enum matters as it is used how to show elements in UI in some cases, where it is sorted by ID
+   */
   type AppealType = Value
   val ADR, REVIEW, APPEAL_TIER_1, APPEAL_TIER_2, COURT_OF_APPEALS, SUPREME_COURT = Value
```

Comment:
Just an idea, but could we avoid making enumeration order significant by using a sealed abstract class or something like that?

```
sealed abstract class AppealType(appealTier: Int) extends Product with Serializable

case object ADR extends AppealType(0)

// etc.
```

Reply:
I tried to change the enums in project to be strictly defined but from/to json start to have a lot of issues this is why I just added comment for now

Reply:
Ah yeah I forgot that play-json doesn't support sealed trait / sealed abstract class